test/unit/configure/test_solver_config.cpp

```diff
@@ -16,6 +16,20 @@ TEST(SolverConfig, NoConfigFilesFound)
   EXPECT_EQ(micm::ConfigParseStatus::NoConfigFilesFound, status);
 }
 
+TEST(SolverConfig, UnknownKey)
+{
+  micm::SolverConfig solverConfig{};
+  auto status = solverConfig.ReadAndParse("./unit_configs/CAMP/camp_unknown_key/config.json");
+  EXPECT_EQ(micm::ConfigParseStatus::UnknownKey, status);
+  try
+  {
+    micm::SolverParameters solver_params = solverConfig.GetSolverParams();
+  }
+  catch (...)
+  {
+  }
```

Comment:
I think it'd be nice to specify `catch` block with `std::runtime_error` since we know what will be thrown